content/post/2017-05-14-robjhyndman-site.md

```diff
@@ -0,0 +1,72 @@
+---
+date: 2017-05-15
+title: Converting robjhyndman to blogdown
```

Comment:
Could you add at least the two fields `author` and `description`? An example: https://github.com/rbind/support/blame/master/content/post/2017-04-25-an-introduction-to-yihui-website.md